gen/dcos-config.yaml

```diff
@@ -367,22 +367,31 @@ package:
       }
 {% case "false" %}
 {% endswitch %}
-  - path: /etc/mesos-slave-dns.json 
+  - path: /etc/mesos-slave-dns.json
     content: |
       {
         "docker":
         [
             {
               "network_mode": "USER",
               "dns": {
-                "nameservers": [ 
+                "nameservers": [
                                 "198.51.100.1",
                                 "fd01:d::c633:6401"
                 ]
               }
             }
         ]
       }
+  - path: /etc/mesos-slave-image-gc-config.json
+    content: |
+      {
+        "image_disk_headroom": 0.1,
+        "image_disk_watch_interval": {
+          "nanoseconds": 300000000000
```

Comment:
Why is this in ns when the default value is going to be so large? Wouldn't "seconds" have been sufficient?

Reply:
This was discussed in the protobuf design patch: https://reviews.apache.org/r/64265/#comment270885

tl;dr: Due to the tech debt, we don't have a sufficient way to represent time duration in protobuf.

From operator's perspective, `nanoseconds` is inconvenient but accurate.